en-US/B.xml

```diff
@@ -229,7 +229,16 @@
 				 <para>
 					According to the <citetitle>IBM Style Guide</citetitle>, it is acceptable to use "boolean" in API programming information when it refers to a primitive return type.
 				</para>
-
+				 <para>
+					To set Boolean values in YAML files, use <varname>true</varname> or <varname>false</varname>, written lowercase, rather than <varname>yes</varname> or <varname>no</varname>, because YAML&nbsp;1.2 and later versions do not support the latter syntax.  
```

Comment:
I think this is fine. We'll probably eventually have guidance that aligns with `ansible-navigator lint` or some of those things that touch upon Ansible.